core/src/test/java/apoc/StartupTest.java

```diff
@@ -1,48 +0,0 @@
-package apoc;
-
-import apoc.util.Neo4jContainerExtension;
-import apoc.util.TestContainerUtil.ApocPackage;
-import apoc.util.TestContainerUtil;
-import apoc.util.TestUtil;
-import org.junit.Test;
-import java.util.List;
-import org.neo4j.driver.Session;
-
-import static apoc.util.TestContainerUtil.createEnterpriseDB;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
-
-/*
- This test is just to verify if the APOC are correctly deployed
- into a Neo4j instance without any startup issue.
- If you don't have docker installed it will fail, and you can simply ignore it.
- */
-public class StartupTest {
-    @Test
-    public void test() {
-        try {
-            Neo4jContainerExtension neo4jContainer = createEnterpriseDB(List.of(ApocPackage.CORE), !TestUtil.isRunningInCI())
-                    .withNeo4jConfig("dbms.transaction.timeout", "60s");
-
-
-            neo4jContainer.start();
-
-            Session session = neo4jContainer.getSession();
-            int procedureCount = session.run("SHOW PROCEDURES YIELD name WHERE name STARTS WITH 'apoc' RETURN count(*) AS count").peek().get("count").asInt();
-            int functionCount = session.run("SHOW FUNCTIONS YIELD name WHERE name STARTS WITH 'apoc' RETURN count(*) AS count").peek().get("count").asInt();
-            int coreCount = session.run("CALL apoc.help('') YIELD core WHERE core = true RETURN count(*) AS count").peek().get("count").asInt();
-
-            assertTrue(procedureCount > 0);
-            assertTrue(functionCount > 0);
-            assertTrue(coreCount > 0);
```

Comment:
I got rid of this test because there's another `StartupTest` and they seem to do the same